roles/artemis/templates/application-prod.yml.j2

```diff
@@ -77,6 +77,11 @@ artemis:
       user: {{ user_management.jira.user }}
       password: {{ user_management.jira.password }}
       admin-group-name: {{ user_management.jira.admin_group}}
+	password-reset:
```

Comment:
The whole block should be in a conditional block. Only if the ansible variables are set this block should be added.